PHPCI/Model/Base/ProjectBase.php

```diff
@@ -148,6 +151,9 @@ class ProjectBase extends Model
             'type' => 'int',
             'length' => 11,
         ),
+        'archive' => array(
+            'type' => 'boolean',
```

Comment:
I didn't know about db:generate, I've added it by hand. Where I can read more about it?